src/components/MAPI/organizations/AccessControl/AccessControlRoleSubjects.tsx

```diff
@@ -146,6 +146,53 @@ const mapValueToSetItem = (stateValue: IStateValue) => (
   };
 };
 
+const formatAccountNames = (accountNames: string[]): string => {
+  return accountNames
+    .map((accountName) => `<code>${accountName}</code>`)
+    .join(', ');
+};
+
+const getBindServiceAccountSuccessMessages = (
```

Comment:
Changed this to return an array of strings - only the non-empty strings will result in a flash message